app/lib/efile/line_data.yml

```diff
@@ -166,6 +166,14 @@ NYC_EIC_RATE_WK_LINE_5:
   label: 'Enter the decimal from the NYC EIC rate table.'
 NYC_EIC_RATE_WK_LINE_6:
   label: 'Subtract line 4 from line 5 or enter the decimal from the NYC EIC rate table. Also enter on Worksheet C, line 2.'
+ID40_LINE_6A:
+  label: '6a Yourself'
+ID40_LINE_6B:
+  label: '6b Spouse'
+ID40_LINE_6C:
+  label: '6c Dependents'
+ID40_LINE_6D:
+  label: '6d Total household'
```

Comment:
could these be a little more descriptive, something like" "primary claimed as dependent", "spouse claimed as dependents", "dependents count" and "total dependent count"

Reply:
ohhh thats a good call i'll fix this!